louiseA/file-1.js

```diff
@@ -0,0 +1,103 @@
+1.
+
+ a) "1" == 1
+  true
+
+ b) "1" === 1
+  false
+
+ c) x == "x"
+  true
+
+ d) x == (x+'')
+  false
+
+ e) '' == ' '
+  true
+
+ f) x = true
+  true
+
+ g) var x; x == 'undefined'
+  false
+
+ h) '9' > '10'
+  true
+
+ i) typeof x + 1 === 'number'
+  false
+
+ j) typeof x % 2 === 'number'
+  false
+
+ k) typeof (x % 2) === 'number'
+  true
+
+ l) x++ == ++x //having trouble understanding the increment operator...
+
+ m) ++x == x++ //see above
+
+ n) "1"+x == 1+x
+  false, unless x is a string
+
+ o) "0" +1 == 1
+ true
+
+ p) (typeof (x+1)) === (typeof x)
+  true, if x is a number
+
+ q) (x*1 == x) || ((typeof x) != "number")
+  true, if x is a number
+
+ r) (x=(typeof (x+(typeof x)))) == x
+  false
+
+
+2.
+
+ a) x+y+z/3
+
+ b) (x/2)+(y/2)+(z/2)
+
+3.
+
+  a) (l+r)(t+b)
+
+  b) (t,b) > (l+r)
+
+  c) /*I know the rules for finding circumference/area of circles, but
+  I'm struggling with getting the variables right if that makes sense */
+
+  d) //see above
+
+  e) //see above
+
+4.
+
+ a) /*I feel like there may be something I'm missing here because I came up with
+ an exression that worked for the '7' column, but when I tried it elsewhere it
+ wouldn't work and I had to adjust the patterns and I didn't solve what was
+ going on but here's the expressions I did come up with... */
+  7s column:
+  ((R+1)C ) + R))
+  5s column:
+  (((R+1)C)=x)+(x/2)
+
```

Comment:
Hey Louise, the answer is actually a lot simpler than that, but you're on the right track. Here is Dan's answer, and I'll go ahead and explain it below: n = 8*r + c;

Firstly, I'm not sure if you drew a picture of the checkerboard or not, but this really helps! I recommend doing this whenever you can when it comes to coding. So basically, we want to find the number ("N") that corresponds with the chosen Row and Column (R, C). So, if we used (0,1) for the Row and Column, the solution would be N= (8*0) +1, in which N equals 1 - which would be correct, because (0,1) is in spot Number 1 on the checkerboard.